script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -41,7 +41,7 @@ My antibodies can render secret arts like Runes and Kabbalah useless, yet someho
 [sha:b362b3538cab1cf4a3f2e755bd1255a823859c06]{
 -- Page 566, Offset 7092.
 -- 「いえ、それだってあそこまでわたしを“殺して”おく事はできない。
-"No, even those wouldn't be able to go so far as to outright 'kill' me.
+"No, even those wouldn't be able to go so far as to outright %{i}kill%{/i} me.
```

Comment:
I'd prefer single quotes here, italics are more like emphasis but quotes indicate that the meaning is different to usual imo.